test/fixd.spec.js

```diff
@@ -0,0 +1,40 @@
+// Setup
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+
+// Source
+const fixd = require('../fixd')
+
+describe('fixd', () => {
+  describe('add', () => {
+    it('throws an error if attempting to assign to already reserved namespace', () => {
```

Comment:
Note that frozen objects only throw in `'use strict'` pragma environments.  This is the default for a Node module as well, however, it won't _always_ throw, such as when used in a browser without `'use strict'` set.